doc/interop-test-descriptions.md

```diff
@@ -90,11 +90,11 @@ Client asserts:
 * clients are free to assert that the response payload body contents are zero
   and comparing the entire response message against a golden response
 
-### large_compressed_unary
+### server_compressed_unary
 
-This test verifies compressed unary calls succeed in sending messages. It
-sends one unary request for every payload type, with and without requesting a
-compressed response from the server.
+This test verifies compressed server-only unary calls succeed in sending
```

Comment:
What is a "server-only call"? You care about compressing messages (not calls), but a unary RPC can't be server-only (it must have a request). How about something more like: "This test verifies the server can compress unary messages."